codespell_lib/data/dictionary.txt

```diff
@@ -6742,6 +6747,11 @@ goin->going
 goind->going
 gonig->going
 gonna->going to, disabled because one might want to allow informal spelling
+gool->ghoul
+gord->gourd
+gormay->gourmet
+gorry->gory
+gost->ghost
```

Comment:
Russian standard (like ISO, EN, DIN, ASME, AISI, ...)
But maybe too special case and can be ignored manually

Reply:
Sorry, I don't understand.

Reply:
From a quick Google, @arm-in means https://en.wikipedia.org/wiki/GOST . I've not seen it before and I'm aware of most of the others, so probably rather niche.